init-exporter.go

```diff
@@ -67,13 +67,14 @@ const (
 	FORMAT_SYSTEMD = "systemd"
 )
 
-// CONFIG_FILE contains path to config file
-const CONFIG_FILE = "/etc/init-exporter.conf"
+// DEFAULT_CONFIG_FILE contains path to config file
+const DEFAULT_CONFIG_FILE = "/etc/init-exporter.conf"
```

Comment:
Using this constant is wrong way. You can just set default value for config argument:
```go
ARG_CONFIG:    &arg.V{Value:"/etc/init-exporter.conf"},
```
In this case, if `--config` is not present, `arg.GetS(ARG_CONFIG)` return path to default value.